### PR TITLE
[T4A6][T16-B4] Yip Bing You

### PR DESCRIPTION
**Pull request by leaffriend**

### CODE REVIEW

**Comment by leaffriend** [comment]
Ready for Review

File: src/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+
+import org.junit.Test;
+
+public class UtilsTest {
```
Comment (lohvht-zz) [inline src/seedu/addressbook/common/UtilsTest.java:10]: missing header comment

File: src/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+	@Test
+	public void nullPresent_returnTrue() {
```
Comment (lohvht-zz) [inline src/seedu/addressbook/common/UtilsTest.java:13]: Note that our naming convention for test methods is as follows:
`[feature under test/method name]_[scenario]_[expected result]`
e.g. `isAnyNull_emptyList_falseReturned`

**Comment by lohvht-zz** [comment]
@leaffriend added comments, please close this PR after you have read the comments

**Pull request closed without merging**